File: URI/1893/1893.c
#include<stdio.h>

int main(){ 

    int pri, seg;
    scanf( "%d%d", &pri, &seg );

	if( 3 <= seg && seg <= 96 && seg > pri ){
     printf( "crescente\n" );
    }
	else if( 3 <= seg && seg <= 96 && seg < pri ){ 
        printf( "minguante\n" );
    }    
	else if( 0 <= seg && seg <= 2 ){
         printf( "nova\n" );	
    }
    else if( 97 <= seg && seg <= 100 ){
         printf( "cheia\n" );
    }
    
    return 0;
}

File: URI/1893/1893.py
pri , seg = map(int , input().split())

if( 3 <= seg and seg <= 96 and seg > pri ):
    print( "crescente" )
    
elif( 3 <= seg and seg <= 96 and seg < pri ):
    print( "minguante" )
    
elif( 0 <= seg and seg <= 2 ):
    print( "nova" )	
    
elif( 97 <= seg and seg <= 100 ):
    print( "cheia" )